pkg/apis/apps/v1beta2/zz_generated.defaults.go

```diff
@@ -176,6 +176,9 @@ func SetObjectDefaults_DaemonSet(in *v1beta2.DaemonSet) {
 			}
 		}
 	}
+	if in.Spec.UpdateStrategy.RollingUpdate != nil {
+		SetDefaults_RollingUpdateDaemonSet(in.Spec.UpdateStrategy.RollingUpdate)
```

Comment:
This should be removed. 

`SetDefaults_DaemonSet` was called in L45 which sets default RollingUpdateDaemonSet already (and sets MaxUnavailable to 1).

Reply:
@janetkuo Updated. PTAL.